README.md

```diff
@@ -204,28 +204,22 @@ The following are tutorials on KubeLinter written by users. If you have one that
 * [How to add a new check in KubeLinter?](https://www.psaggu.com/upstream-contribution/2021/08/17/notes.html) by Priyanka Saggu.
 * [Extending kube-linter To Build A Custom Template](https://github.com/garethahealy/kubelinter-extending-blog) by [Gareth Healy](https://github.com/garethahealy).
 
-## Community
-
-If you would like to engage with the KubeLinter community, including maintainers and other users, you can join the Slack workspace [here](https://join.slack.com/t/kube-linter/shared_invite/zt-kla9qvyo-Tk~wynTSbr9EE3AjHcv4BQ).
-
-To contribute, check out our [contributing guide](./CONTRIBUTING.md).
-
-As a reminder, all participation in the KubeLinter community is governed by our [code of conduct](./CODE_OF_CONDUCT.md).
+## LICENSE
 
-## WARNING: Alpha release
+KubeLinter is licensed under the [Apache License 2.0](./LICENSE).
 
-KubeLinter is at an early stage of development. There may be breaking changes in
-the future to the command usage, flags, and configuration file formats. However,
-we encourage you to use KubeLinter to test your environment YAML files, see what
-breaks, and [contribute](./CONTRIBUTING.md).
+## Community
 
-## LICENSE
+If you would like to engage with the KubeLinter community, including maintainers and other users, you can join the Slack workspace [here](https://join.slack.com/t/kube-linter/shared_invite/zt-kla9qvyo-Tk~wynTSbr9EE3AjHcv4BQ).
 
-KubeLinter is licensed under the [Apache License 2.0](./LICENSE).
+There may be breaking changes in the future to the command usage, flags, and configuration
+file formats. However, we encourage you to use KubeLinter to test your environment YAML
+files, see what breaks, and [contribute](./CONTRIBUTING.md).
 
-## StackRox
+As a reminder, all participation in the KubeLinter community is governed by our
+[code of conduct](./CODE_OF_CONDUCT.md).
 
-KubeLinter is made with ❤️ by [StackRox](https://stackrox.com/).
+##
```

Comment:
Do we want a title for this section ? Or is the goal here to just have a separating line in the rendered file format ?

Reply:
If the lather then `---` should do the trick

Reply:
`---` is not in the [syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) but I can try that.

Reply:
Worked, thanks!
<img width="355" alt="Screenshot 2024-11-29 at 13 24 23" src="https://github.com/user-attachments/assets/8aa25857-9f2f-42ca-800c-3431ab7eb93c">